src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -288,10 +288,9 @@ export const selectPortfolioAllocationPercentByFilter = createSelector(
     const balanceAllocationById = Object.entries(assetFiatBalancesByAccount).reduce<{
       [k: AccountSpecifier]: number
     }>((acc, [currentAccountId, assetAccountFiatBalance]) => {
-      const allocation = bnOrZero(assetAccountFiatBalance[assetId])
-        .div(bnOrZero(totalAssetFiatBalance))
-        .times(100)
-        .toNumber()
+      const allocation = bnOrZero(
+        bn(assetAccountFiatBalance[assetId]).div(bnOrZero(totalAssetFiatBalance)).times(100)
+      ).toNumber()
```

Comment:
I just found out that dividing by 0 in bignumber.js results in `Infinity`. I wonder if we should update `bnOrZero` to treat `Infinity` and `-Infinity` the same as `NaN`?

```suggestion
      const allocation = bnOrZero(bn(assetAccountFiatBalance[assetId]).div(totalAssetFiatBalance).times(100)).toNumber()
```

Reply:
Yes I agree. I'm going to use `isFinite` instead of the `isNaN` we currently use to check for Infinity and -Infinity